conanfile.py

```diff
@@ -0,0 +1,116 @@
+from conans import ConanFile, CMake, tools
+import sys
+
+class NovelRTConan(ConanFile):
+    name = "NovelRT"
+    version = "0.1"
+    license = "MIT"
+    url = "https://github.com/novelrt/NovelRT"
+    description = "A cross-platform 2D game engine accompanied by a strong toolset for visual novels."
+    settings = "os", "compiler", "build_type", "arch"
+    requires = [
+        ("freetype/2.10.1"),
+        ("glad/0.1.33"),
+        ("glfw/3.3.2"),
+        ("glm/0.9.9.7"),
+        ("gtest/1.10.0"),
+        ("libsndfile/1.0.30"),
+        ("openal/1.19.1"),
+        ("spdlog/1.8.2"),
+        ("ms-gsl/3.1.0"),
+        # Commented until Vulkan support is introduced.
+        #("vulkan-loader/1.2.172"),
+        #("vulkan-memory-allocator/2.3.0")
+    ]
+    generators = "cmake_find_package", "cmake_paths"
+    options = {
+        "inksupport": [True, False],
+        "documentation": [True, False],
+        "buildtests": [True, False],
+        "buildsamples": [True, False]
+    }
+    default_options = {
+        "freetype:shared":True,
+        "glfw:shared":True,
+        "libsndfile:shared":True,
+        "openal:shared":True,
+        "PNG:shared":True,
+        "BZip2:shared":True,
+        "flac:shared":True,
+        "fmt:shared":True,
+        "Opus:shared":True,
+        "Ogg:shared":True,
+        "Vorbis:shared":True,
+        "glad:gl_profile":"core",
+        "glad:gl_version":4.0,
+        "glad:gles2_version":3.0,
+        "spdlog:header_only":True,
+        #"vulkan-loader:shared":True,
```

Comment:
Why is this commented out?

Reply:
It's not currently a part of `main` branch and causes issues if the library required isn't being included (see the Vulkan deps that are commented as well). I only added anything pertaining to Vulkan as we'll eventually need it (aka when you pull main into your graphics branch 🙂 )

Reply:
What kind of issues? That sounds wonky, but thanks for the heads up.

Reply:
Well, if I uncomment the option but not the dep, python gets upset because the option doesn't exist (as a key) to set the value. At least that's what I was seeing on my box.

(DW, uncommenting both option and dep made everything happy w/o issue)

Reply:
Ohhh I see. Makes sense then. Must've missed the dependency comment.